lib/patchelf/exceptions.rb

```diff
@@ -0,0 +1,9 @@
+# encoding: ascii-8bit
+# frozen_string_literal: true
+
+require 'elftools/exceptions'
+
+module PatchELF
+  # Raised on an error during ELF modification.
+  class PatchError < ELFError ; end
```

Comment:
prepend `ELFTools::` to ELFError to have more precise semantic

Reply:
Thanks for the catch!